pkg/controller/download_controller.go

```diff
@@ -100,6 +100,17 @@ func (c *downloadController) enqueueDownloadItem(obj interface{}) {
 	switch req.Status.Phase {
 	case "", pluginv1api.DownloadPhaseNew, pluginv1api.DownloadPhaseInProgress, pluginv1api.DownLoadPhaseRetry:
 		// Process New InProgress and Retry Downloads
+	case pluginv1api.DownloadPhaseCompleted:
+		// If Download CR status reaches terminal state more than 1 hour, Download CR should be deleted
+		now := c.clock.Now()
+		if req.Status.CompletionTimestamp.Add(time.Hour).Unix() < now.Unix() {
```

Comment:
Let's flip this to
now > req.Status.CompletionTimestamp.Add(time.Hour)
it's easier to figure out what it's doing. Do we need to use Unix() for the compare? If we do, put it in, but I don't think it's necessary.

Reply:
Same as above.